.ci/Jenkinsfile

```diff
@@ -74,7 +74,9 @@ pipeline {
       }
       steps {
         withCredentials([string(credentialsId: "${GITHUB_TOKEN_CREDENTIALS}", variable: 'GITHUB_TOKEN')]) {
-          sh 'curl -sL https://git.io/goreleaser | bash'
+          dir("${BASE_DIR}") {
+            sh 'curl -sL https://git.io/goreleaser | bash'
```

Comment:
Is it possible to store this file in this repo?

Reply:
I can do this, I assume it's potential security threat.

Maybe we can even further and install it together with `withGoEnv`? WDYT?